web/src/i18n/locales/en.json

```diff
@@ -65,7 +65,7 @@
     "nominateSpeaker": {
         "bio": {
             "hint": "A brief description of the nominee",
-            "label": "Speaker Bio"
+            "label": "Speaker Bio written in the third person (he/she/they, etc.)"
```

Comment:
Sorry, only caught this now, but maybe we should use placeholder or hint field? Otherwise the error message sounds a bit odd. What do you think?

<img width="917" alt="Screen Shot 2020-09-05 at 7 46 15 PM" src="https://user-images.githubusercontent.com/1096046/92303526-99182380-efb0-11ea-87ad-5fec09ddfbb2.png">